### DISCUSSION TITLE
Control order of box sections

### DISCUSSION THREAD

**Discussion by PubliAlex** [OFFLINE-GmbH/oc-boxes-support#19] [Ideas and feature requests]
Hello,

I wonder how boxes sections are ordered by default, it doesn't seems to be by name... Would it be possible to get control of the order of boxes sections ?

Best regards,

Alex

**Comment by tobias-kuendig** [comment]
Hi @PubliAlex 

We have an issue pending for this. An idea we currently are looking at is the introduction of an optoinal `boxes.yaml` config file. In there, you could define the order of your sections:

```yaml
sections:
- General
- E-Commerce
- CTA
```

What do you think, would that be a solution for your use-case?

Ref #20

**Comment by PubliAlex** [comment]
@tobias-kuendig yeah, it would be perfect

**Comment by tobias-kuendig** [comment]
This is now available in v2.0.91:
https://docs.boxes.offline.ch/use-cases/editor-customization.html#changing-the-order-of-partial-sections
